horace_core/sqw/@sqw/private/split_npix_full_bins.m

```diff
@@ -0,0 +1,83 @@
+function [npix_chunks, idxs] = split_npix_full_bins(npix, page_size, num_pix, npix_cum_sum)
+%SPLIT_NPIX_FULL_BINS Split the given array of counts such that the sum of each
+%  sub-array has a maximum of page_size.
+%
+% The npix counts are not split across bin boundaries - i.e. each sub-array
+% contains only full bins. If a value in npix is greater than page_size, then
+% that value will comprise its own sub-array.
+%
+% Input:
+% ------
+% npix          An array of integers (usually sqw.data.npix)
+% page_size     An integer specifying the maximum sum for each sub-array
+% num_pix       The sum of npix [optional].
+%               If this argument is not given it is calculated on-the-fly.
+%               In most cases, this value is cached for sqw objects in
+%               sqw.data.pix.num_pixels.
```

Comment:
If it's stored, is this attached to the object? Can it just be retrieved?

Reply:
Not in the case of cut. The output of  `proj.get_nbin_range` in [`cut_accumualte_data_`](https://github.com/pace-neutrons/Horace/pull/610/files#diff-b1ef44eea1c119f36c058dfaedcf05f4b743713d5fca2d1b905424a7e6ee982cR43) corresponds to bin edges for the output cut. So we have an `npix` array, but it's not attached to an object yet